tests/core/policies/test_ted_policy.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+from typing import List, Text
+
+import numpy as np
+import pytest
+from rasa.core.policies.ted_policy import TEDPolicy
+from rasa.shared.core.constants import ACTION_LISTEN_NAME
+from rasa.shared.core.domain import Domain
+from rasa.shared.core.events import ActionExecuted, UserUttered
+from rasa.shared.core.trackers import DialogueStateTracker
+from rasa.shared.nlu.interpreter import RegexInterpreter
+
+UTTER_GREET_ACTION = "utter_greet"
+GREET_INTENT_NAME = "greet"
+DOMAIN_YAML = f"""
+intents:
+- {GREET_INTENT_NAME}
+actions:
+- {UTTER_GREET_ACTION}
+"""
+
+
+def test_diagnostics():
```

Comment:
I think it's great to have a separate module for this 🚀  Could you please move the remaining tests for the `TEDPolicy` from `test_policies`  to this module?